docs/cli-help.txt

```diff
@@ -81,7 +81,9 @@ optional arguments:
                         agent strings allow the CA to collect high level
                         statistics about success rates by OS and plugin. If
                         you wish to hide your server OS version from the Let's
-                        Encrypt server, set this to "". (default: None)
+                        Encrypt server, set this to "". (default:
+                        CertbotACMEClient/0.10.0.dev0 (Debian GNU/Linux
```

Comment:
Is this what we want default to say? Presumably this will be regenerated once we update the version number so it doesn't say `0.10.0.dev0`?